src/elements/toniq-input/toniq-input.element.ts

```diff
@@ -134,6 +134,7 @@ export const ToniqInput = defineToniqElement({
 
         label {
             flex-grow: 1;
+            width: 100%;
```

Comment:
What was wrong without this change? Please add an example to the storybook to showcase the problem before and the fix afterwards.